about.html

```diff
@@ -62,17 +62,17 @@
         <div class="row">
           <div class="col-12">
               <nav class="navbar navbar-expand-lg mobile-nav-wrapper">
-                <a class="navbar-brand" href="#">
+                <a class="navbar-brand" href="index.html">
                     <img class="logo-custom-mobile" src="./assets/image/logo.png" alt="" srcset="">
                 </a>
                 <div class="" id="">
                   <ul class="navbar-nav">
                     <li class="nav-item active">
                       <a class="nav-link" href="index.html">Home </a>
                     </li>
-                    <li class="nav-item">
+                    <!-- <li class="nav-item">
                       <a class="nav-link" href="#">Courses</a>
-                    </li>
+                    </li> -->
                     <li class="nav-item">
                       <a class="nav-link" href="about.html">About</a>
                     </li>
```

Comment:
- [ ] On the Mobile view of the about page, I noticed that you're missing the Partners section. So kindly implement that in your Design 👍 
- [ ] Also, please style the contents of the footer as seen in the original design with the logo aligned to the left and text aligned to the right. 👍 
- Your Design
<img width="288" alt="Screen Shot 2023-05-24 at 11 36 08 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/03ecdb42-7623-467c-80c1-d0de3555760f">


- The Behance Design
<img width="387" alt="Screen Shot 2023-05-24 at 11 37 51 AM" src="https://github.com/Aqurds/wd-course/assets/102699958/b8bed835-8361-43b2-b350-452ab6b21f8f">